OpenRA.Mods.D2k/Traits/Carryable.cs

```diff
@@ -51,7 +52,9 @@ public Carryable(Actor self, CarryableInfo info)
 		public void MovingToResources(Actor self, CPos targetCell, Activity next) { RequestTransport(targetCell, next); }
 		public void MovingToRefinery(Actor self, CPos targetCell, Activity next) { RequestTransport(targetCell, next); }
 
-		void RequestTransport(CPos destination, Activity afterLandActivity)
+		public int MinimumDistance { get { return info.MinDistance; } }
+
+		public void RequestTransport(CPos destination, Activity afterLandActivity, Actor ignoreActor = null)
 		{
 			if (destination == CPos.Zero || (self.Location - destination).Length < info.MinDistance)
```

Comment:
You need to be careful about quantization when comparing against cell lengths.  I suggest subtracting cell-centers instead.